test/safe/pull-request.test.ts

```diff
@@ -647,7 +646,7 @@ describe("Pull Request Webhook", () => {
 							branches:
 								[
 									{
-										createPullRequestUrl: "test-pull-request-head-url/compare/TEST-321-test-pull-request-head-ref?title=TEST-123%20TEST-321%20TEST-124%20-%20TEST-321-test-pull-request-head-ref&quick_pull=1",
+										createPullRequestUrl: useNewGithubClient ? "test-pull-request-head-url/compare/TEST-321-test-pull-request-head-ref?title=TEST-123%20TEST-321%20TEST-124%20-%20TEST-321-test-pull-request-head-ref&quick_pull=1" : "test-pull-request-head-url/pull/new/TEST-321-test-pull-request-head-ref",
```

Comment:
createPullRequestUrl has its own independent FF you could use use-new-github-pull-request-url-format